core/sail/nativerdf/src/main/java/org/eclipse/rdf4j/sail/nativerdf/TxnStatusFile.java

```diff
@@ -77,25 +94,55 @@ public void close() throws IOException {
 
 	/**
 	 * Writes the specified transaction status to file.
-	 * 
+	 *
 	 * @param txnStatus The transaction status to write.
 	 * @throws IOException If the transaction status could not be written to file.
 	 */
 	public void setTxnStatus(TxnStatus txnStatus) throws IOException {
-		byte[] bytes = txnStatus.name().getBytes(US_ASCII);
+		byte[] bytes = txnStatus.onDisk;
```

Comment:
Does that work now that you've made it private? I would have expected you need to use `getOnDisk()` here instead.

Reply:
Was surprised too. Seems that this is legal. No compile issues.